WebService/appsettings.ini

```diff
@@ -31,6 +31,32 @@ webservice_url = "${PCS_STORAGEADAPTER_WEBSERVICE_URL}"
 # Max time allowed for requests to the storage adapter before reporting an error
 webservice_timeout = 10000
 
+[AzureManagementService]
+# URL where the azure management api is available.
+webservice_url = "https://management.azure.com"
+
+# Max time allowed for requests to the azure management service before reporting an error.
+# Value in milliseconds
+webservice_timeout = 10000
+
+# Api version
+api_version = "2015-11-01"
```

Comment:
I'd like to see these prefixed with something indicating that these configs are related to the AzureManagementService, something like: azure_management_webservice_url, etc. When I 'webservice_url', it isn't immediately clear to me that you're referring to the URL of the Azure Management service.

Reply:
Please refer to `/WebService/Runtime/Config.cs`.